realm/realm-library/src/objectServer/java/io/realm/UserStore.java

```diff
@@ -68,4 +68,12 @@
      * @return Collection of all users. If no users exist, an empty collection is returned.
      */
     Collection<SyncUser> allUsers();
+
+    /**
+     * Returns the state of the specified user: {@code true} if active (not logged out), {@code false} otherwise.
+     *
+     * @param identity identity of the user.
+     * @return {@code true} if the user is not logged out, {@code false} otherwise.
+     */
+    boolean isActive(String identity);
```

Comment:
Wouldn't it be better for this to accept a `SyncUser`? This is the abstraction that people normally work with? (Also the rest of the methods in the interface)